contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -289,34 +289,92 @@ NXmpes(NXobject):
       (NXmanipulator):
         exists: optional
         doc: |
-          Manipulator for positioning of the sample.
-        sample_temperature(NX_FLOAT):
+          Manipulator for positioning of the sample.      
+        temperature_sensor(NXsensor):
           exists: recommended
-          unit: NX_TEMPERATURE
-        drain_current(NX_FLOAT):
-          exists: recommended
-          unit: NX_CURRENT
-        sample_bias(NX_FLOAT):
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [temperature]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+            exists: recommended
+        sample_heater(NXactuator):
           exists: recommended
-          unit: NX_CURRENT
-          doc:
-          - |
-            Bias of the sample with respect to analyser ground.
-            
-            This field may also be found in NXsample if present.
-          - |
-            xref:
-              spec: ISO 18115-1:2023
-              term: 8.41
-              url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:8.41
-        device_information(NXfabrication):
```

Comment:
Why has the device_information been removed?

Reply:
Accidental. Has been readded to NXmanipulator.

The xref part above has been added to NXsample/bias again.